types/tsconfig.json

```diff
@@ -0,0 +1,57 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "commonjs",
+    "lib": ["esnext"],
+    // "allowJs": true,
```

Comment:
I think the commented out compiler options can be removed.

Reply:
Ok, I'll remove.